src/main/flight/pid.c

```diff
@@ -917,7 +915,7 @@ void FAST_CODE pidController(const pidProfile_t *pidProfile, const rollAndPitchT
         float acErrorRate;
 #endif        
 
-        const float ITerm = pidData[axis].I;
+        const float iterm = pidData[axis].I;
```

Comment:
it should be iTerm

Reply:
"Like all the others"??  Everywhere in the entire code it is `iterm`, not ITerm, except in config_unittest.h, and for the total of iterm in pid.c.
Everywhere else these are the conventions, e.g.:
`itermAcceleratorGain` and `pidResetIterm` now follow camel case rules strictly; or
`iterm_relax`, iterm all in lower case between underscores; or
`ITERM_RELAX_SETPOINT` i.e all caps
There is no place where the first two characters are capitalised, not pterm or dterm either, again except config.unittest.h.
The best consistency would be to uncapitalise config_unittest.h for ITerm PTerm etc back to iterm, pterm, etc; then we have proper consistency everywhere.

Reply:
Yeah I saw that later. I like this better too.

You can change it in unittest too if you want